package.json

```diff
@@ -40,10 +40,13 @@
   "bin": {
     "cds-typer": "./lib/cli.js"
   },
-  "dependencies": {
-    "@sap/cds": ">=7.7"
+  "peerDependencies": {
```

Comment:
I can not come up with any scenario on the spot, but are there use cases where there is no `@sap/cds` directly available? CI/CD? Within the vscode extension? 🤔

Reply:
> Within the vscode extension?

Good point.  Is cds-typer installed w/ our extension?  Don't we rely on the app installing it?  In this case, we are fine.